lib/options/option_parser.rb

```diff
@@ -54,8 +62,16 @@ def parse(args)
               'BrowserStack device to use (a key of Devices.DEVICE_HASH)',
               short: :none,
               type: :string
-          opt Option::USERNAME, 'Device farm username', short: '-u', type: :string
-          opt Option::ACCESS_KEY, 'Device farm access key', short: '-p', type: :string
+          opt Option::USERNAME,
+              'Device farm username',
+              short: '-u',
+              type: :string,
+              default: ENV['MAZE_DEVICE_FARM_USERNAME']
```

Comment:
This does work, but the help doesn't tell the user that they can use the environment variable.  We may need to explicitly change the descriptions to something like "Device farm username (defaults to MAZE_DEVICE_FARM_USERNAME)".

Reply:
Added some extra detail to the help strings.